decentraland/unity-renderer
# feat: update enums for ecs7

## What does this PR change?

This PR updates the proto version to include enums.

Since the enum also had the same name as the unity enum, I also added a namespace for that enum

## How to test the changes?


1. Go to: https://play.decentraland.zone/?renderer-branch=feat/update-proto-version
2. Check that the camera behave as expected

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
